yb-voyager/src/dbzm/dbzm.go

```diff
@@ -25,26 +25,24 @@ func init() {
 	} else {
 		DEBEZIUM_DIST_DIR = "/opt/yb-voyager/debezium-server"
 	}
-
-	DEBEZIUM_CONF_DIR = filepath.Join(DEBEZIUM_DIST_DIR, "conf")
-	DEBEZIUM_CONF_FILEPATH = filepath.Join(DEBEZIUM_CONF_DIR, "application.properties")
 }
 
 func NewDebezium(config *Config) *Debezium {
 	return &Debezium{Config: config}
 }
 
 func (d *Debezium) Start() error {
+	DEBEZIUM_CONF_FILEPATH = filepath.Join(d.ExportDir, "metainfo/conf/application.properties")
```

Comment:
Make it a local variable then:

    dbzmConfFilePath := filepath.Join(...)

Reply:
Seems like this var is being used by another func, will need to keep it global.